src/jvm/test/org/jetbrains/kotlinx/lincheck_test/gpmc/ModelCheckerTest.kt

```diff
@@ -101,6 +101,6 @@ private class CollectResultsVerifier : Verifier {
     }
 }
 
-private const val DEFAULT_INVOCATIONS_COUNT = 100
+private const val DEFAULT_INVOCATIONS_COUNT = 500
```

Comment:
Please revert

Reply:
This change is necessary. 
Tests in `gpmc` subpackage run a lambda, collect all possible outcomes of this lambda into set, and compare it with the expected set:
https://github.com/JetBrains/lincheck/blob/966b6213bb6f48c2b881a69e9b9e0d4a7b5e64a3/src/jvm/test/org/jetbrains/kotlinx/lincheck_test/gpmc/MonitorTest.kt#L39

When the local object detector is disabled, the model checker might need to explore more interleavings in order to collect all possible outcomes.